resolwe/flow/models/entity.py

```diff
@@ -29,6 +29,9 @@ class Meta(BaseCollection.Meta):
     #: indicate whether `descriptor` is completed (set by user)
     descriptor_completed = models.BooleanField(default=False)
 
+    #: tags for categorizing objects
+    tags = ArrayField(models.CharField(max_length=255), default=list)
```

Comment:
Passing a set will not change anything. We can add a special field which transforms this on save (but you can't guarantee this at the database level).